packages/ddp-client/common/livedata_connection.js

```diff
@@ -551,6 +551,27 @@ export class Connection {
     return this.apply(name, args, callback);
   }
 
+  /**
+   * @memberOf Meteor
+   * @importFromPackage meteor
+   * @alias Meteor.callAsync
+   * @summary Invokes asynchronously a method passing any number of arguments.
+   * @locus Anywhere
+   * @param {string} name Name of method to invoke
+   * @param args Optional method arguments
+   */
+  async callAsync(name, ...args) {
```

Comment:
No need to use the `async` keyword here, since you are already returning a promise.